Monika After Story/game/script-story-events.rpy

```diff
@@ -539,6 +539,10 @@ label unlock_chess:
     m 3hua "But I have a new game for us to play!"
     m "This one's a lot more strategic..."
     m 3hub "It's Chess!"
+    if persistent._mas_pm_likes_board_games = False :
```

Comment:
This will crash when getting to this line.

Also, we'll use `is` here for a bit more clarity (and to keep up with the rest of the coding style)
```suggestion
    if persistent._mas_pm_likes_board_games is False :
```